src/ephemeris.rs

```diff
@@ -36,7 +36,8 @@ type Result<T> = std::result::Result<T, InvalidEphemeris>;
 
 /// Various statuses that an ephemeris can be in
 #[derive(Copy, Clone, Debug)]
-#[repr(u32)]
+#[cfg_attr(windows, repr(i32))]
+#[cfg_attr(not(windows), repr(u32))]
```

Comment:
Maybe we should instead switch to `repr(C)`? Based on this in the reference: https://doc.rust-lang.org/reference/type-layout.html#reprc-field-less-enums 

It sounds like we should maybe abandon this approach in the long term since it risks invoking UB and isn't scalable to particularly weird target/compiler configs.

Reply:
Using `repr(C)` fails with the same issue unfortunately

Reply:
Unfortunate. Oh well